README.md

```diff
@@ -72,6 +83,26 @@ Here are some examples:
 
 ```
 
+If you want to run the same commands but for linux environment, you must change the script_stepwise_phase123.bat file with script_stepwise_phase123_unix.sh
+
+If you want to run an evaluation in Alliance Canada servers (or possibly other servers that support job running using sbatch) use the following commands:
```

Comment:
Explicitly point out that only phase 0 and 2 require GPU as model inference is involved. For other steps, no GPU, this only CPU is needed. Then you can mention for phase 0 and 2, these scripts from alliance can be used.